modules/test/playwright/pages/object-web/ViewObjectDefinitionsPage.ts

```diff
@@ -33,9 +35,15 @@ export class ViewObjectDefinitionsPage {
 		this.defaultObjectFolderLink = page
 			.locator('li')
 			.filter({hasText: 'Default'});
+		this.objectFolders = page
+			.getByRole('list')
+			.filter({hasText: 'Default'});
```

Comment:
i didn't understand very well what are you trying to do here, do you want to get the default folder only or all the folders present on the page?

Reply:
The element being selected is the list, but I might receive more than one list, and in this case I am filtering to specify that I want the list that contains the text "Default" in its content. Therefore, I am receiving the list of all folders.

Reply:
oh i see now! the selectors used do make this association very intuitive, but i understand now thanks!